README.md

```diff
@@ -1 +1,177 @@
-# https://www.kaggle.com/c/h-and-m-personalized-fashion-recommendations
\ No newline at end of file
+# **laplace**, a Graph Neural Network based Recommendation Engine
+
+##### Table of Contents  
+- [Overview](#Overview)  
+- [Get Started](#Get_Started) 
+  - [Installation & Data](#Installation) 
+  - [Main Pipeline](#Main_Pipeline) 
+- [Advanced Usage](#Advanced_Usage)  
+    - [Hyperparameter tuning](#Hyperparameter_tuning)     
+- [Further todo](#Further_todo)  
+- [Recommended resources](#Recommended_resources)  
+
+<br>
+
+
+
+<a name="Overview"></a>
+
+## Overview
+
+**laplace** is an end-to-end ML framework to train and predict on neurally-enhanced graphs for recommendation. 
+
+The pipeline is designed for self-supervised edge prediction on heterogenous graphs. 
+
+<br>
+
+![Pipeline Overview](/docs/assets/pipeline_overview.png)
+
+### Features
+
+- Two step, **Multi-model** composition:
+  1. Candidate Selection
+     - Integrating LightGCN recommendations (can be ran on its own aswell)
+     - Multiple, custom heuristics
+     - Strategies can be mixed and matched
+  2. GraphConvolutional network prediction on 
+- Works on **Heterogenous** graphs
+  - User based training, validation and test splitting
+  - N-hop neighborhood aggregation
+  - Node Features 
+  - Works on any number of node types
+- Advanced preprocessing of tabular data into graphs
+  - **Neo4j** integration for better visualization and handling of large graphs.
+
+
+
+
+<br>
+<br>
+
+<a name="Get_Started"></a>
+
+## Get Started
+
+
+### Installation & Data
+Install the environment with:
+
+    conda env create -n fashion --file environment.yml
```

Comment:
`conda env create` automatically searcher for environment.yml and creates it (with the correct name) :)